sass/includes/_buttons.scss

```diff
@@ -13,6 +13,11 @@
         font-size: 0.7rem;
     }
 
+    &__icon {
```

Comment:
We already have support for icons within buttons (https://nationalarchives.github.io/tna-frontend/?path=/story/components-button--icon) - is this required?

Reply:
Good spot, I've removed this since updating the icons to font awesome.